GliaWidgets/SecureConversations/ChatTranscript/SecureConversations.TranscriptModel+GVA.swift

```diff
@@ -118,6 +118,7 @@ private extension SecureConversations.TranscriptModel {
                     in: self.pendingSection
                 )
             case .failure:
+                environment.log.prefixed(Self.self).info("Show Unexpected error Dialog")
```

Comment:
would it make sense to log error also?

Reply:
Yes. But has to be done on both platforms.